lib/bpk-component-text/src/BpkText-test.android.js

```diff
@@ -22,11 +22,12 @@ import React from 'react';
 import renderer from 'react-test-renderer';
 
 import BpkThemeProvider from '../../bpk-theming';
-import { WEIGHT_STYLES } from '..';
 
 import BpkText from './BpkText';
 import commonTests from './BpkText-test.common';
 
+import { WEIGHT_STYLES } from '..';
```

Comment:
Can we have `../` instead? Feels more explicit

Reply:
Sure

Reply:
Actually no, I tried doing it and it reverts back to `..` when I save (and the linter/prettier runs). Sorry :P

Reply:
I think I had an old version of the linter installed because this change was reverted once I did `npm i` re-ran it.